emission/storage/timeseries/cache_series.py

```diff
@@ -58,7 +58,7 @@ def insert_entries(uuid, entry_it):
             ts.insert(entry)
             tsdb_count = tsdb_count + 1
         else:
-            ucdb.save(entry)
+            ucdb.insert_one(entry)
```

Comment:
can you revert this change, or replace it with `edb.save(ucdb, entry)`? `save` != `insert`, and I want to be careful about making a change to the core functionality in `emission/storage/`.

Reply:
Ok I reverted it.